tests/integration_tests.py

```diff
@@ -52,21 +52,24 @@ def test_create_model_from_model_package(self, **kwargs):
     @mock_sagemaker_session
     def test_compare_metrics(self, **kwargs):
         sm_client = kwargs.pop("sm_client")
-        endpoint_config_description = sm_client.describe_endpoint_config(EndpointConfigName=self.endpoint_name)
-        evaluate_s3 = 's3://sagemaker-mlops-p-sq33brxtdjtx/EvaluateAbaloneModel-f4800dc7aed96399c5afac406ea9afe0/output/evaluation/evaluation.json'
-        metric_path = 'regression_metrics/mse/value'
-        compare_metrics(sagemaker_client=sm_client,
-                        endpoint_config_description=endpoint_config_description,
-                        model_statistics_s3_uri=evaluate_s3,
-                        metric=metric_path,
-                        dryrun=True
-                        )
+        endpoint_config_description = sm_client.describe_endpoint_config(
+            EndpointConfigName=self.endpoint_name
+        )
+        evaluate_s3 = "s3://sagemaker-mlops-p-sq33brxtdjtx/EvaluateAbaloneModel-f4800dc7aed96399c5afac406ea9afe0/output/evaluation/evaluation.json"
```

Comment:
We can simply drop that variable or replace it with an empty string since it's not used in `compare_metrics` whatsoever 🤷